src/hvm.rs

```diff
@@ -2403,23 +2405,22 @@ pub fn name_to_u128(code: &str) -> u128 {
   return num;
 }
 
-// Inverse of name_to_u128
+/// Inverse of `name_to_u128`
 pub fn u128_to_name(num: u128) -> String {
   let mut name = String::new();
   let mut num = num;
   while num > 0 {
     let chr = (num % 64) as u8;
-    if chr == 0 {
-      name.push('.');
-    } else if chr < 10 {
-      name.push((chr + 0 + '0' as u8) as char);
-    } else if chr < 36 {
-      name.push((chr - 11 + 'A' as u8) as char);
-    } else if chr < 62 {
-      name.push((chr - 37 + 'a' as u8) as char);
-    } else if chr == 63 {
-      name.push('_');
-    }
+    let chr =
+        match chr {
+            0         => '.',
+            1  ..= 10 => (chr -  1 + b'0') as char,
+            11 ..= 36 => (chr - 11 + b'A') as char,
+            37 ..= 62 => (chr - 37 + b'a') as char,
+            63        => '_',
+            64 ..     => panic!("impossible character value")
```

Comment:
This version is even better as one does not have to comment out the wildcard arm to see the aforementioned useful error message.